lib/Dialect/RTL/RTLOps.cpp

```diff
@@ -459,6 +459,66 @@ static LogicalResult verifyInstanceOp(InstanceOp op) {
       return failure();
   }
 
+  // If the referenced moudle is internal, check that input and result types are
+  // consistent with the referenced module.
+  if (isa<RTLModuleOp>(referencedModule)) {
```

Comment:
As per the llvm style guide prefer early exits. e.g. if not rtlmoduleop, return.